docs/deployment/observability/logs.mdx

```diff
@@ -58,7 +58,7 @@ Take care when enabling `trace` or `debug` level logs in a production environmen
 
 In a production environment, it's more common to emit logs in a structured format and aggregate logs from all hosts in a centralized location. Enabling OTEL signals automatically structures JSON logs for OTEL export, while local `stderr` logs remain plaintext.
 
-To enable structured log output _without_ enabling OTEL signals, use the `--enable-structured-logging` [host config](/docs/reference/host-config) argument. This will cause the host to emit logs as JSON, which are suitable for forwarding to a log aggregator.
+To enable structured log output _without_ enabling OTEL signals, use the `--enable-structured-logging` [host config](/docs/cli/wasmcloud) argument. This will cause the host to emit logs as JSON, which are suitable for forwarding to a log aggregator.
```

Comment:
```suggestion
To enable structured log output _without_ enabling OTEL signals, use the `--enable-structured-logging` [host config](/docs/cli/wasmcloud) argument (or set the `WASMCLOUD_STRUCTURED_LOGGING_ENABLED` environment variable). This will cause the host to emit logs as JSON, which are suitable for forwarding to a log aggregator.
```